api/core/model_runtime/model_providers/wenxin/llm/ernie-speed-8k.yaml

```diff
@@ -0,0 +1,30 @@
+model: ernie-speed-8k
+label:
+  en_US: ERNIE-Speed-8K
+model_type: llm
+features:
+  - agent-thought
+model_properties:
+  mode: chat
+  context_size: 8192
+parameter_rules:
+  - name: temperature
+    use_template: temperature
+    min: 0.1
+    max: 1.0
+    default: 0.95
+  - name: top_p
+    use_template: top_p
+    min: 0
+    max: 1.0
+    default: 0.7
+  - name: max_tokens
+    use_template: max_tokens
+    default: 1024
+    min: 1
+    max: 8000
```

Comment:
max_output_tokens of `ernie-speed-8k` is limited to 1024, refer to https://cloud.baidu.com/doc/WENXINWORKSHOP/s/klqx7b1xf